backend/app/tests/api/routes/collections/test_create_collections.py

```
@@ -1,97 +1,49 @@
-import pytest
 from uuid import UUID
-import io
+from unittest.mock import patch
 
-from sqlmodel import Session
 from fastapi.testclient import TestClient
 from unittest.mock import patch
```

Ruff (0.13.1)

Redefinition of unused `patch` from line 2

Remove definition: `patch`

(F811)